lib/common/logger.js

```diff
@@ -57,6 +59,13 @@ function loggerFactory(
         assert.isIn(level, levels);
         assert.string(message, 'message');
 
+        // Exit if the log level of this message is less than the minimum log level.
+        var minLogLevel = nconf.get('minLogLevel');
+        if ((minLogLevel === undefined) || (typeof minLogLevel !== 'number')) {
+            minLogLevel = 0;
+        }
+        if (Constants.Logging.Levels[level] < minLogLevel) return;
```

Comment:
We try to avoid braceless if statements as a style rule, if you wouldn't mind changing this.